ironfish-rust/src/mining/thread.rs

```diff
@@ -130,12 +127,12 @@ fn process_commands(
                         }
                     }
 
-                    batch_start += default_batch_size;
-                    if batch_start >= MAX_SAFE_INTEGER {
+                    if remaining_search_space < batch_size as u64 {
```

Comment:
Also, should this be `==` rather than `<`? I saw up above that if `remaining_search_space <= default_batch_size`,`batch_size` gets set to `remaining_search_space`, so I'm not sure if there's a case where it'll be less.

Might be misreading, but just wanted to double-check the logic here

Reply:
Ah, you're absolutely correct. I meant for this to be `default_batch_size` instead of `batch_size`